index.html

```diff
@@ -0,0 +1,120 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <link rel="stylesheet" href="style.css" />
+  <title>
+    Document
+  </title>
+</head>
+
+<body>
+  <div class=border>
+    <h1> Jaye Lin Resume </h1>
+    <div class="contact">
+      <p> <a href="Illusxia. carbonmade.com"> Portfolio </a><br>
+
+        <a href="Xiaoxxia23@Gmail.com"> Email </a> <br>
+        Baltimore, MD
+      </p>
+    </div>
+
+    <div class="edu">
+      <h3> Education And Training: </h3>
+      <p>
+        <b>Maryland Institute Of Art And Design</b>
+        <br />
+        <i>(2019-Current)</i><br />
+        Bachelor of graphic design
+      </p>
+
+      <p>
+        <b>University Of Minnesota- Twin Cities</b><br />
+        <i>(2014-2018)</i><br />
+        Bachelors of fine arts<br />
+        Deans list
+      </p>
+
+      <p>
+        <b> Animation mentor</b><br />
+        <i>(2018)</i><br />
+        Character animation
+      </p>
+    </div>
+
+    <div class="expert">
+
+      <h3>Expertise</h3>
+      <p><b>Coding</b>: Css, Html<br />
+        <b>Programs</b>: Rhino, Maya, Photoshop, After Effects, Indesign,
+        Illustrator<br />
+        <b>Skills</b>: Sketching, Illustration, Drafting, Character Design,
+        Visual/World Development
+      </p>
```

Comment:
Consider treating this as a list; it's a bit more CSS to achieve the same layout, but it's more much semantically meaningful:
```suggestion
      <ul>
            <li>Coding:
                <ul>
                    <li>CSS</li>
                    <li>HTML</li>
                </ul>
            </li>
            <li>Programs:
                <ul>
                    <li>Rhino</li>
                    <li>Maya</li>
                    <li>Photoshop</li>
                    <li>After Effects</li>
                    <li>InDesign</li>
                    <li>Illustrator</li>
                </ul>
            </li>
            <li>Skills:
                <ul>
                    <li>Sketching</li>
                    <li>Illustration</li>
                    <li>Drafting</li>
                    <li>Character Design</li>
                    <li>Visual/World Development</li>
                </ul>
            </li>
      </ul>
```